packages/compiler-core/__tests__/transforms/vIf.spec.ts

```diff
@@ -520,6 +520,16 @@ describe('compiler: v-if', () => {
       )
     })
 
-    test.todo('with comments')
+    test('with comments', () => {
+      const { node } = parseWithIfTransform(`<div v-if="ok // Test comment"/>`)
```

Comment:
This is not what this test is meant to test. The case is when there are HTML comments between `if` and `else` branches.